docs/tutorials/reward_networks.rst

```diff
@@ -0,0 +1,131 @@
+
+There are two types of wrapper:
+* :class:`ForwardWrapper <imitation.rewards.reward_nets.ForwardWrapper>` allows for direct modification of the results of the reward network's ``forward`` method. It is used during the learning of the reward network and thus must be differentiable. These wrappers are always applied first and are always active. They are used for applying transformations like potential shaping (see :class:`ShapedRewardNet <imitating.rewards.reward_nets.ShapedRewardNet>`).
+* :class:`PredictProcessedWrapper <imitation.rewards.reward_nets.PredictProcessedWrapper>` modifies the predict_processed call to the reward network. Thus this type of reward network wrapper is designed to only modify the reward when it is being used to train/evaluate a policy but *not* when we are taking gradients on it. Thus it does not have to be differentiable.
+
+The most commonly used is the :class:`NormalizedRewardNet <imitating.rewards.reward_nets.NormalizedRewardNet>` which is a predict procssed wrapper. This class uses a normalization layer to standardize the *output* of the reward function using its running mean and variance, which is useful for stabilizing training. When a reward network is saved, its wrappers are saved along with it, so that the normalization fit during reward learning can be used during future policy learning or evaluation.
+
+.. testcode::
+    :skipif: skip_doctests
+
+    from imitation.rewards.reward_nets import NormalizedRewardNet
+    from imitation.util.networks import RunningNorm
+    train_reward_net = NormalizedRewardNet(
+        reward_net,
+        normalize_output_layer=RunningNorm,
+    )
+
+.. note::
+    The reward normalization wrapper does _not_ function identically to stable baselines3's `VecNormalize <https://stable-baselines3.readthedocs.io/en/master/guide/vec_envs.html#stable_baselines3.common.vec_env.VecNormalize>`_ environment wrapper. First, it does not normalize the observations. Second it normalizes the reward based on the reward networks's mean and variance on and *not* a running estimate of the return.
+
+By default, the normalization wrapper updates the normalization on each call to ``predict_processed``. This behavior can be altered as shown below.
+
+.. testcode::
+
+    from functools import partial
+    eval_rew_fn = partial(reward_net.predict_processed, update_stats=False)
+
+Serializing and Deserializing Reward Networks
+---------------------------------------------
+
+Reward networks are serialized simply by calling ``th.save(reward_net, path)``. When evaluating reward networks, we may or may not want to include the wrappers it was trained with. To load the raw reward network, we can simply call ``th.load(path)``.
+
```

Comment:
On the first pass reading it was not clear here whether `th.save(reward_net, path)` will also save the wrappers and if `th.load(path)` would restore them. I then read the code to clarify and came back to the description to read it a second time. Then it was clear.

Maybe you can help (quick) understanding by adding to the first sentence:

"Reward networks and **their wrappers** are serialized simply by calling ..."

and 

"to laod the raw reward network **without wrappers applied**, we can simply call ..."